src/ActionGroup.php

```diff
@@ -0,0 +1,76 @@
+<?php
+declare (strict_types=1);
+
+namespace LotGD\Core;
+
+/**
+ * A grouping of navigation actions, like a submenu.
+ */
+class ActionGroup
+{
+    private $id;
+    private $title;
+    private $sortKey;
+    private $actions;
+
+    /**
+     * Create a new ActionGroup, starting with an empty set of actions.
+     * @param string $id Unique identifier for this group, in the vendor/module/group format.
+     * @param string $title Title to display to the end user. Empty string means no title.
+     * @param string $sortKey Navigation menus are displayed in the order sorted by this string.
```

Comment:
Oh, I never thought about this aspect.

Though I still think that integers or floats are the better decision - it's easier to "see" that 15 < 20, or 0.123 < 0.135, instead of comparing "AAAB" to "AAA". Though floats may produce unforseen results if rounding comes to the issue.

Multiples of 10 or 100 is probably the better, yeah. Maybe decimals would be even better, though they cannot be used in PHP directly.